critiquebrainz/frontend/static/styles/main.less

```diff
@@ -490,3 +490,22 @@ a#edit-review { margin-top: 20px; }
     margin-top: 25px;
   }
 }
+
+// Moderator page
+
+#list {
+  margin-top: 30px;
+}
+#list img {
+  width: 25px;
+  height: 25px;
+}
+
+#list a {
+  margin-right: 3px;
+}
+
+#list p {
+  display: inline-block;
+  margin-right: 3px;
+}
```

Comment:
`list` ID is a bit too generic though, I suggested an alternative in another comment.